example/stochastic_subgrid/stochastic_subgrid_package.cpp

```diff
@@ -319,8 +319,8 @@ void DoLotsOfWork(MeshBlockData<Real> *rc) {
           for (int n = 0; n < num_vars; ++n)
             (n % 2 == 0 ? even : odd) += sqrt(n + 1) * v(in + n, k, j, i);
 
-          Real a = pow(10.0, (odd + even) / (fmax(1.0, abs(odd * even))));
-          Real b = pow(10.0, (odd - even) / (fmax(1.0, abs(odd * even))));
+          Real a = pow(10.0, (odd + even) / (fmax(1.0, std::abs(odd * even))));
+          Real b = pow(10.0, (odd - even) / (fmax(1.0, std::abs(odd * even))));
```

Comment:
Why are we just doing this for the abs methods?

Reply:
Because someone might come along and remove the std namespace I would add a comment, or I would call fabs instead.

Reply:
updated to fabs